Dockerfile

```diff
@@ -3,8 +3,8 @@ FROM hhvm/hhvm:4.128.2
 RUN apt update -y
 RUN DEBIAN_FRONTEND=noninteractive apt install -y php-cli zip unzip openssh-client
 
-RUN curl -sS https://getcomposer.org/installer | php -- --install-dir=/usr/local/bin --filename=composer
+COPY --from=composer:latest /usr/bin/composer /usr/local/bin/composer
 ENV HHVM_VERSION=4.128.2
 
-ADD . /app
 WORKDIR /app
+COPY . /app
```

Comment:
Just curious - I'm not a Docker expert, but if we use `COPY` will changes I make during local development be reflected in the container?

Reply:
Ohh me neither, but I do know they should be equivalent in that regard. For context https://stackoverflow.com/a/24958548